lms/djangoapps/courseware/grades.py

```diff
@@ -126,9 +127,18 @@ def grade(student, request, course, keep_raw_scores=False):
     """
     Wraps "_grade" with the manual_transaction context manager just in case
     there are unanticipated errors.
+    Send a signal to update the minimum grade requirement status.
     """
     with manual_transaction():
-        return _grade(student, request, course, keep_raw_scores)
+        grade_summary = _grade(student, request, course, keep_raw_scores)
+        MIN_GRADE_REQUIREMENT_STATUS.send_robust(
```

Comment:
If I'm reading the [documentation](https://docs.djangoproject.com/en/1.8/topics/signals/) for `send_robust` correctly, it catches exceptions and returns them.  I think it would make sense to log them here.